src/openstack_cloud/openstack_manager.py

```diff
@@ -345,6 +350,22 @@ def reconcile(self, quantity: int) -> int:
 
         return delta
 
+    def _reconcile_reactive(self, quantity: int) -> int:
+        """Reconcile runners reactively.
+
+        Args:
+            quantity: Number of intended runners.
+
+        Returns:
+            The difference between intended runners and actual runners. In reactive mode
+            this number is never negative as additional processes should terminate after a timeout.
+        """
+        logger.info("Reactive mode is experimental and not yet fully implemented.")
+        config = reactive_runner_manager.ReactiveRunnerConfig(
+            mq_uri=self._config.reactive_config.mq_uri, queue_name=self.app_name
+        )
+        return reactive_runner_manager.reconcile(quantity=quantity, config=config)
```

Comment:
Think about whether the config is needed or it can be passed as arguments directly